src/splits.rs

```diff
@@ -1680,7 +1692,13 @@ pub fn transition_splits(
                 && scenes.current == "Song_Enclave",
         ),
         Split::TrobbioTrans => should_split(mem.deref(&pd.defeated_trobbio).unwrap_or_default()),
-        //endregion: ChoralChambers
+        // endregion: ChoralChambers
+
+        // region: Underworks
+        Split::PostClawlineArenaTrans => {
+            should_split(scenes.old == "Under_18" && scenes.current == "Under_17")
```

Comment:
There are two transitions between `Under_18` and `Under_17`, which could be distinguished by the `entryGateName` field of the `GameManager` class, through `entry_gate_name` in `GameManagerPointers`. I'd recommend rebasing your branch on master before you try to make this change